### ISSUE TITLE
Feature Request: Paste on Select in Clipboard Indicator

### ISSUE THREAD

**Issue by arvindchauhan1** [Tudmotu/gnome-shell-extension-clipboard-indicator#476]
**Automated Paste Action**: Implement a feature where selecting content in the clipboard indicator automatically triggers a paste action into the active application or text field.

**Customizable Settings**: Provide users with the ability to customize this behavior according to their preferences. Options could include enabling/disabling the automatic paste action and adjusting the delay before pasting.

**Comment by thinh105** [comment]
Yes, I would love this feature, especially since it would replicate the behavior seen in clipboard management on Windows OS, where selection results in automatic pasting

**Comment by youyoumu** [comment]
I also need this

**Comment by MichalVasut** [comment]
I'm also all hands fot it - it would be super  usefull and improves the usability.

**Comment by robertvb** [comment]
would love to have that

**Comment by Tudmotu** [comment]
Fixed in #503 , uploaded to e.g.o
